example/src/Examples/WithBottomSheet.tsx

```diff
@@ -0,0 +1,59 @@
+import BottomSheet, { BottomSheetFlatList } from '@gorhom/bottom-sheet';
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+import React from 'react';
+import {
+  SafeAreaView,
+  KeyboardAvoidingView,
+  Platform,
+  StyleSheet,
+  Button,
+} from 'react-native';
+import { RichText, Toolbar, useEditorBridge } from '@10play/tentap-editor';
+
+export const WithBottomSheet = ({}: NativeStackScreenProps<any, any, any>) => {
+  const bottomSheetRef = React.useRef<BottomSheet>(null);
+  const editor = useEditorBridge({
+    autofocus: true,
+    avoidIosKeyboard: true,
+    initialContent,
+  });
+
+  const openBottomSheet = () => {
+    bottomSheetRef.current?.expand();
+  };
+
+  const closeBottomSheet = () => {
+    bottomSheetRef.current?.close();
+  };
+
+  return (
+    <SafeAreaView style={exampleStyles.fullScreen}>
+      <Button title="Open Bottom Sheet" onPress={openBottomSheet} />
+      <BottomSheet snapPoints={['70%']} ref={bottomSheetRef}>
+        <Button title="Close Bottom Sheet" onPress={closeBottomSheet} />
+        <RichText editor={editor} />
+        <KeyboardAvoidingView
+          behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+          style={exampleStyles.keyboardAvoidingView}
+        >
+          <Toolbar editor={editor} ListComponent={BottomSheetFlatList} />
```

Comment:
On IOS the toolbar is not visible

![Screenshot 2024-07-29 at 14 12 40](https://github.com/user-attachments/assets/0e8144a8-a7b9-4f33-9ca9-baa6e3851e91)

On android we are unable to scroll inside the text editor

https://github.com/user-attachments/assets/90517dd8-14c7-4fc1-af40-4ddf87b31af9

Reply:
@17Amir17 I have updated the bottom sheet example. You can review it again


https://github.com/user-attachments/assets/cac17beb-4744-4eee-bec5-094896215003


https://github.com/user-attachments/assets/14c6b680-8b5d-4a8e-ab70-5f950e066bfd

Reply:
@17Amir17 I have updated the PR. You can re-review it.

iOS:

https://github.com/user-attachments/assets/37878093-b9aa-4703-b62f-9781f5b328bd

Android:

https://github.com/user-attachments/assets/a68dc1ef-9db5-4224-9695-6dd3502a4540